.rspec

```diff
@@ -0,0 +1,3 @@
+--format progress
+--color
+--require spec_helper
```

Comment:
You can also add `--warnings`

Reply:
Done in https://github.com/thoughtbot/appraisal/pull/250/commits/a19e70746cce6bdab027e914c62a497033af181c